package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@netcentric/stylelint-config",
   "description": "Netcentric's coding and style rules for Stylelint",
-  "version": "1.2.1",
+  "version": "1.3.0",
```

Comment:
Package version is changed automatically after a release, no need to change it manually.
Please revert this change.

Reply:
done